content/en/platform/corda/5.0/developing-applications/packaging/cpk-plugin.md

```diff
@@ -0,0 +1,422 @@
+---
+date: '2023-06-06'
+title: "CPK Gradle Plugin"
+project: corda
+version: 'Corda 5.0'
+menu:
+  corda5:
+    parent: corda5-develop-packaging
+    identifier: corda5-develop-packaging-cpk-plugin
+    weight: 4000
+section_menu: corda5
+---
+# CPK Gradle Plugin
+
+Applying this plugin to a project declares that the project should create a CorDapp in CPK format. The CPK format
+CorDapp is a ZIP file with a `.jar` extension, which is the output of the JAR task, along with the dependent JARs of that JAR.
+The plugin will not include any of Corda's own JARs among these dependencies, nor any JARs that should be provided by
+Corda, such as Kotlin or Quasar. The JAR should also contain sufficient OSGi metadata to be a valid OSGi bundle.
+
+## Applying the Plugin
+
+1. Depending on the number of modules of your Gradle project, choose one of the following options:
+
+* If your Gradle project contains multiple modules, perform the following actions:
```

Comment:
Can't see a preview but it looks like there should be indentation here for the second-level bullet points to work?

Reply:
Corrected the indentation.